RecoLocalTracker/SiPixelRecHits/plugins/BuildFile.xml

```diff
@@ -1,18 +1,28 @@
 <use name="fmt"/>
-<use name="CUDADataFormats/BeamSpot"/>
-<use name="CUDADataFormats/TrackingRecHit"/>
-<use name="HeterogeneousCore/CUDACore"/>
+<use name="CondFormats/SiPixelTransient"/>
+<use name="DataFormats/BeamSpot"/>
+<use name="DataFormats/TrackerCommon"/>
+<use name="DataFormats/TrackingRecHitSoA"/>
+<use name="HeterogeneousCore/AlpakaCore"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
```

Comment:
I'd expect `AlpakaCore` (and maybe `AlpakaInterface`) dependence to apply only to `RecoLocalTrackerSiPixelRecHitsPluginsPortable`.